controllers/controllers/networking/cfroute_controller.go

```diff
@@ -461,10 +461,6 @@ func (r *CFRouteReconciler) fetchServicesByMatchingLabels(ctx context.Context, l
 	return &serviceList, nil
 }
 
-func isFinalizing(cfRoute *korifiv1alpha1.CFRoute) bool {
```

Comment:
Why remove the helper function? I feel like it provides some quicker context to what `!cfRoute.GetDeletionTimestamp().IsZero()` means. But maybe could be renamed to `isBeingDeleted` or `markedForDeletion` or something. 🤷